manifests/base/common/configmap.yaml

```diff
@@ -74,8 +74,10 @@ data:
         command:
         - /usr/local/bin/chaos
         args:
+        - --no-log-file
```

Comment:
please remove this from this PR, would require another one if to be modified, thanks